client/containers/DashboardReviews/DashboardReviews.tsx

```diff
@@ -16,6 +16,13 @@ const DashboardReviews = (props: Props) => {
 	const { pubsWithReviews } = props;
 	const { scopeData } = usePageContext();
 	const { activeCollection, activeTargetType } = scopeData.elements;
+	const parseReviewers = (reviewers: Reviewer[] | undefined) => {
```

Comment:
Rereading my comment above, I misread 'in-competent' three times before I could get it right.

Reply:
lmao, i think it can stay bc otherwise i would have to set reviewers on the server, which isnt a bad idea. but since we pull in reviews and iterate there i think its easier to leave them